src/server/index.jsx

```diff
@@ -40,6 +40,13 @@ class LoggerStream {
 
 const server = express();
 
+server.use((req, res, next) => {
+  const urlPath = /\?[^]*\//.test(req.url);
```

Comment:
Do we need `urlPath`? https://amitd.co/code/javascript/strip-trailing-slashes-from-express-requests as an example doesnt use it?

Also can you move this to `if (process.env.APP_ENV === 'local') {` please, as it should be local only :)